vcpkg/VCPKG_BOOST_LOCALE_DEPS_LIST

```diff
@@ -0,0 +1,15 @@
+boost-assert:
+boost-build:
+boost-config:
+boost-function:
+boost-integer:
+boost-iterator:
+boost-modular-build-helper:
+boost-smart-ptr:
+boost-static-assert:
+boost-system:
+boost-thread:
+boost-type-traits:
+boost-unordered:
+boost-vcpkg-helpers:
+libiconv:
```

Comment:
Why libiconv is put into VCPKG_BOOST_LOCALE_DEPS_LIST ? How it is related to BOOST deps?

Reply:
This file contains a list of `boost-locale` dependencies (you can check them at https://github.com/microsoft/vcpkg/blob/master/ports/boost-locale/CONTROL). The actual reason why this list even exists is that `boost-locale` is broken - can't be built under sanitizers.
So the whole idea is next: build all dependencies under sanitizers, then build `boost-locale` without them and then build the rest with sanitizers again.